partner_firstname/models/base_config_settings.py

```diff
@@ -61,7 +61,10 @@ def action_recalculate_partners_name(self):
         )
         partners = self._partners_for_recalculating()
         _logger.info("Recalculating names for %d partners.", len(partners))
-        partners._compute_name()
+        # Use add_to_compute instead of _compute_name to avoid triggering
+        # _inverse_name_after_cleaning_whitespace, which can
```

Comment:
why the inverse it's triggered?

Reply:
This is because the `name` field (which is now a stored field) has as its inverse the `inverse_name_after_cleaning_whitespace` method:

https://github.com/OCA/partner-contact/blob/23c2d101e08c36f3909091c5f488110ce45e14cc/partner_firstname/models/res_partner.py#L23

so when calling the `_compute_name` method directly, its inverse is called afterwards:

https://github.com/odoo/odoo/blob/72a5da62d00feb247e4dcc5e6be52955de1c83f8/odoo/models.py#L3808

When using `add_to_compute` instead, the `name` is recomputed during `flush` here:

https://github.com/odoo/odoo/blob/72a5da62d00feb247e4dcc5e6be52955de1c83f8/odoo/fields.py#L1370

 and the `write` method is not called*, so the inverse is not triggered.
 
 *Note, I am unsure of why the write method is not called, I believe it has to do with the `_compute_name` method being [called as a function](https://github.com/odoo/odoo/blob/72a5da62d00feb247e4dcc5e6be52955de1c83f8/odoo/fields.py#L1343) with arguments instead of as a method belonging to the `Field` class, which might prevent other methods from `self` from triggering.